Makefile

```diff
@@ -76,7 +77,9 @@ proto/vendor:
 
 .PHONY: container
 container:
-	buildah build-using-dockerfile --timestamp 0 --layers --build-arg VERSION=$(VERSION) --build-arg COMMIT=$(COMMIT) -t $(OUT_DOCKER):$(VERSION) .
+	for arch in $(ALL_ARCH); do \
+	buildah build-using-dockerfile --build-arg VERSION=$(VERSION) --build-arg COMMIT=$(COMMIT) --build-arg ARCH=$$arch --arch $$arch --timestamp 0 --manifest $(OUT_DOCKER):$(VERSION); \
```

Comment:
One caveat here is that this removes caching of layers. If devs use "make container" as part of their workflow, they might want to readjust (or add a different single-arch target).

Reply:
Good call, can we maybe have a "container-dev" make target for the dev use case?